src/power_udev/provider_power_udev.cpp

```diff
@@ -511,17 +546,19 @@ class BatteryNs : public statefs::Namespace
      *
      * - LowBattery [0, 1] - is battery level below low battery
      *   threshold (defined by BATTERY_LOW_LIMIT) environment variable
+     *   and charging is not going on
      *
      * - TimeUntilLow (sec) - approx. time until battery will be empty
      *
      * - TimeUntilFull (sec) - approx. time until battery will be charged
      *
-     * - Temperature (integer, °C * 10) - battery zone temperature if provided
+     * - Temperature (integer, °C * 10) - battery zone temperature if
+     *   provided
      *
      * - Power (integer, uW) - average power consumed during several
      *   last measurements (positive - charging)
      *
-     * - State (deprecated, string) [unknown, charging, discharging, full, low,
+     * - State (string) [unknown, charging, discharging, full, low,
```

Comment:
yes, there were several use cases where it allows to use only this property instead of ChargingState+Level, so less subscriptions